src/main/java/appeng/services/export/MinecraftItemCSVExporter.java

```diff
@@ -209,76 +210,70 @@ public String apply( @Nullable final ItemStack input )
 
 		@Nullable
 		@Override
-		public String apply( @Nullable final Item input )
-		{
-			if( input == null )
-			{
-				AELog.debug( EXPORTING_NOTHING_MESSAGE );
+		public String apply( @Nullable final Item input ) {
+			if (FMLCommonHandler.instance().getSide().isClient()) {
```

Comment:
Exit early and don't make your IDE reformat the file into your preferred form and we won't have git noise as huge as this one.